caluma/form/schema.py

```diff
@@ -751,6 +751,7 @@ class RemoveAnswer(Mutation):
     class Meta:
         lookup_input_kwarg = "answer"
         serializer_class = serializers.RemoveAnswerSerializer
+        return_field_type = Answer
```

Comment:
Small change, big effect :tada:

Reply:
Unfortunately #382 is a different issue :(

Reply:
Pity. I'll go ahead and approve this one anyway, then.